lib/app_size_report/plugin.rb

```diff
@@ -2,8 +2,23 @@
 
 module Danger
   require 'json'
+  require 'open-uri'
+  require 'fileutils'
+
   require_relative '../converter/parser/report_parser'
   require_relative '../converter/helper/memory_size'
+  require_relative '../converter/helper/android_utils'
+
+  $project_root = Dir.pwd
+  $temp_path = "#{$project_root}/temp"
+  $apks_path = "#{$temp_path}/output.apks"
+  $size_csv_path = "#{$temp_path}/output.csv"
+  $bundletool_path = "#{$temp_path}/bundletool.jar"
+  $bundletool_version = "1.8.2"
+  $variants_limit = 30
+
+  $default_screen_densities = ["MDPI", "HDPI", "XHDPI", "XXHDPI", "XXXHDPI"]
+  $default_languages = ["en"]
 
   # A Danger plugin for reporting iOS app size violations.
```

Comment:
Let's change this description to just say "A Danger plugin used to report iOS and Android app size violations."